packages/core/src/bag3d/core/assets/reconstruction/reconstruction.py

```diff
@@ -127,7 +127,7 @@ def reconstructed_building_models_nl(
     try:
         return_code, output = context.resources.roofer.app.execute(
             exe_name="roofer",
-            command=f"{{exe}} --config {{local_path}} {output_dir} --loglevel {context.op_config['loglevel']} --skip-pc-check --no-tiling",
+            command=f"{{exe}} --config {{local_path}} {output_dir} --loglevel {context.op_config['loglevel']} --skip-pc-check",
```

Comment:
@Ylannl is it ok to remove `--no-tiling` from the roofer command? 
Since we already tile the input and feed the individual tiles to roofer, they shouldn't be split even more.